src/barfTurn.ts

```diff
@@ -338,6 +367,34 @@ const turns: AdventureAction[] = [
     spendsTurn: false,
     sobriety: Sobriety.SOBER,
   },
+  {
+    name: "Pig Skinner Free-For-All",
+    available: () =>
+      have($skill`Free-For-All`) &&
+      !have($effect`Everything Looks Red`) &&
+      romanticMonsterImpossible(),
+    execute: () => {
+      const usingDuplicate = SourceTerminal.have() && SourceTerminal.duplicateUsesRemaining() > 0;
```

Comment:
Given that FFA doesn't force drops, we probably don't want to use duplicate here